frontend/device/android/device.lua

```diff
@@ -109,6 +109,18 @@ function Device:retrieveNetworkInfo()
     end
 end
 
+function Device:update(url, filename)
```

Comment:
If we add a Device:update() there should also be a no-op in generic.

Reply:
moved that to ota manager :+1: